File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SweetAndSourMeat.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Food;

/* JADX INFO: loaded from: classes3.dex */
public class SweetAndSourMeat extends Food {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Consumable, it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.food_sweet_and_sour_meat_name;
        this.idDescription = R.string.food_sweet_and_sour_meat_description;
        this.idImage = R.drawable.sweet_and_sour_meat;
        this.price = 21L;
        this.feedPower = 38;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SweetAndSourMeat.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Food

class SweetAndSourMeat : Food() {
    override fun configureProperties() {
        idName = R.string.food_sweet_and_sour_meat_name
        idDescription = R.string.food_sweet_and_sour_meat_description
        idImage = R.drawable.sweet_and_sour_meat
        price = 21L
        feedPower = 38
    }
}
